client/apollo/js/View/Track/AnnotTrack.js

```diff
@@ -6425,9 +6425,7 @@ define([
 
             executeUpdateOperation: function (postData, loadCallback) {
                 if(!postData.hasOwnProperty('clientToken')){
-                    var postObject = JSON.parse(postData);
-                    postObject.clientToken = this.getClientToken();
-                    postData = JSON.stringify(postObject);
+                    postData.clientToken = this.getClientToken();
```

Comment:
Good catch. Now that `postData` is strictly a JSON object, another `JSON.parse` will definitely fail. Which is what led to the error.